pytorch_object_detection/detection_models/__init__.py

```diff
@@ -0,0 +1,66 @@
+import torchvision
+import torch
+from torchvision.io.image import read_image
+from torchvision.models import WeightsEnum
+
+from torchvision.utils import draw_bounding_boxes
+from torchvision.transforms.functional import to_pil_image
+
+
+class DetectionPipeline:
+    model: torchvision.models.detection
+    weights: WeightsEnum
+    preprocess_pipeline: torchvision.transforms._presets.ObjectDetection
+    model_classes: list
```

Comment:
Move to __init__(...)

Reply:
As only 'model' and 'weights' need to be defined in child class init method  and call parent init to define preprocess_pipeline and model_classes so I added it as hint for defining child init, or it is better only leave  model: torchvision.models.detection
    weights: WeightsEnum

Reply:
Move it

Reply:
done